app.py

```diff
@@ -2,33 +2,12 @@
 Flask Application
 '''
 from flask import Flask, jsonify, request
-from models import Experience, Education, Skill
+from models import Experience, Education, Skill #pylint: disable=unused-import
+from utils import load_data, save_data #pylint: disable=unused-import
 
 app = Flask(__name__)
 
-data = {
-    "experience": [
-        Experience("Software Developer",
-                   "A Cool Company",
-                   "October 2022",
-                   "Present",
-                   "Writing Python Code",
-                   "example-logo.png")
-    ],
-    "education": [
-        Education("Computer Science",
-                  "University of Tech",
-                  "September 2019",
-                  "July 2022",
-                  "80%",
-                  "example-logo.png")
-    ],
-    "skill": [
-        Skill("Python",
-              "1-2 Years",
-              "example-logo.png")
-    ]
-}
+data = load_data('data.json')
```

Comment:
Let's add try-catch here after we have the error returning done in the utils.py

Reply:
To be able to relay the server error to client and also accommodate previous PRs where I believe is the right place and also anticipate the check for the nonexistence of data (where data could be None in the event of an exception)...

I am contemplating between two options. I can either raise the exception, thereby terminating the application's execution (I doubt that's the right approach), or I can include a check to determine if data is None within the /test route.

Or any other ideas? What do you think? Am I on the right track?

Reply:
Good point. For None data raising an exception and termination of program flow is not ideal, let's catch the exception in this case, print the error in console.

For user side, let's return 500 error in case data is None. What do you say?

Reply:
On it

Reply:
Sounds good, did you already create the issue? If not, could you please create one, and we can look into it later